deep_agent/aegra/mcp.py

```
@@ -442,15 +446,15 @@ async def _connect_single_server(
                     "[%s] MCP OAuth required — returning auth placeholder tool",
                     name,
                 )
-                return [_create_auth_placeholder_tool(name)]
+                return [_create_auth_placeholder_tool(name, server_cfg)]
             elif _is_auth_error(exc):
```

Ruff (0.16.1)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)